src/assets/toolkit/styles/functions/index.scss

```diff
@@ -0,0 +1,13 @@
+/* A simple deep-map-get function to load
+nested map values from the yaml or json data files */
+
+@function data($map, $keys...) {
+  $value: $map;
+  @each $key in $keys {
+    $value: map-get($value, $key);
+  }
+  $result: unquote(inspect($value));
+    //  Inspect is used here to silence a warning about depricating
+    //  none-string unquoting in future versions of sass  
+  @return $result;
+}
```

Comment:
Add a `toolkit()` function shortcut that pulls in the toolkit map, eliminating the first argument.

e.g.:
`color:  toolkit(colors, primary, Dark)`